src/components/Blog/Blog.react.js

```diff
@@ -109,6 +95,86 @@ const BlogFooter = styled.div`
   flex-wrap: wrap;
 `;
 
+const CardMedia = styled(_CardMedia)`
+  height: 0;
+  padding-top: 56.25%;
+  height: 500px;
+  object-fit: contain;
+  vertical-align: middle;
+
+  @media (max-width: 1000px) {
+    height: 300px;
+  }
+`;
+
+const Card = styled(_Card)`
+  position: relative;
+`;
+
+const Overlay = styled.div`
+  position: relative;
+  left: 0px;
+  background: rgba(0, 0, 0, 0.54);
+  width: 100%;
+  padding: 1rem;
+  margin-top: -3.5rem;
+`;
+
+const CustomTypography = styled(Typography)`
+  margin-bottom: 2rem;
+  color: rgba(0, 0, 0, 0.54);
+  font-size: 0.875rem;
+`;
+
+const BlogNavigation = styled.div`
+  display: none;
+  justify-content: center;
+  padding-top: 15px;
+
+  @media (min-width: 1400px) {
+    position: absolute;
+    right: 15%;
+    bottom: 220px;
+  }
+
+  @media (max-width: 1000px) {
+    bottom: 200px;
+  }
+`;
+
+const SocialButtons = styled.div`
+  width: 100%;
+  display: flex;
+  padding: 10px 0 20px 10px;
+`;
+
+const Icon = styled.i`
+  padding-right: 10px;
+`;
+
+const Container = styled.div`
+  width: 100%;
+`;
+
+const BlogFooterLink = styled.a`
+  white-space: nowrap;
+`;
+
+const OverlayLink = styled.a`
+  &&& {
+    ${LinkStyle};
```

Comment:
Why?

Reply:
@fragm3 This is done to override the style from `.section-blog a` CSS.